apps/fulfillment-e2e/src/integration/filters.cy.ts

```diff
@@ -0,0 +1,83 @@
+import { FiltersFragment } from '../support/page_fragments/filters.fragment';
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+const filtersFragment = new FiltersFragment();
+
+describe('When user opens the filters', () => {
+  const largeAndEarliestOrderFirst = (exists = true) =>
+    pickingListsFragment
+      .getPickingListsItems()
+      .eq(0)
+      .find('.total oryx-icon-button button')
```

Comment:
All selectors should exist only in page objects or page fragments. 

When we put selectors in spec files - we violate that pattern + we make our tests less stable and increase maintenance time and costs